src/common-components/FormGroup.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import { Form, TransitionReplace } from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+const FormGroup = (props) => {
+  const [hasFocus, setHasFocus] = useState(false);
+
+  const handleFocus = (e) => {
+    setHasFocus(true);
+    if (props.handleFocus) { props.handleFocus(e); }
+  };
+  const handleOnBlur = (e) => {
+    setHasFocus(false);
+    if (props.handleBlur) { props.handleBlur(e); }
+  };
+
+  return (
+    <Form.Group controlId={props.name} className="mb-4" isInvalid={props.errorMessage !== ''}>
+      <Form.Control
+        as={props.as}
+        type={props.type}
+
+        name={props.name}
+        value={props.value}
+        onFocus={handleFocus}
+        onBlur={handleOnBlur}
+        onChange={props.handleChange}
+
+        trailingElement={props.trailingElement}
+        floatingLabel={props.floatingLabel}
+      >
+        {props.options ? props.options() : null}
+      </Form.Control>
+      <TransitionReplace>
+        {hasFocus && props.helpText ? (
+          <Form.Control.Feedback key="help-text" className="mt-2">
+            {props.helpText.map((message) => (
+              <>
```

Comment:
This will raise a warning in the browser console, can we use an actual element here with a key?